test/fixtures/v5/fork/straight_destination.json

```diff
@@ -10,7 +10,7 @@
     },
     "instructions": {
         "ar": "توجّه نحو مباشرة لمسافة {distance} ",
-        "da": "Hold mod ligeud mod Destination 1",
+        "da": "Hold til ligeud mod Destination 1",
```

Comment:
Should be "Fortsæt ligeud mod Destination 1"

Reply:
So if `modifier` is empty, "Hold til" should be replaced with "Fortsæt"?

Reply:
If `modifier` is `straight`, at least.

In English, you use "Keep" in both "Keep left" and "Keep straight".

In Danish you'd have to say "Hold til venstre" and "Fortsæt ligeud", respectively.

Reply:
That's when `modifier` is "ligeud"

Reply:
Yes

Reply:
So as I could understand, only "ligeud (straight)" modifier requires different phrase beginning - "Fortsæt", all others should start with "Hold til", correct? Then it would be better to update Transifex with most used forms - then special post-processing necessity will be reduced.

Reply:
Yes that's correct, and I agree. With "fork", only the `straight` modifier is wrong.